src/Button.test.js

```diff
@@ -0,0 +1,34 @@
+import Button, { StyledButton } from "./Button";
+import { configure, shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import React from "react";
+
+configure({ adapter: new Adapter() });
+
+describe("Button", () => {
+  it("renders a blank button", () => {
+    expect.assertions(1);
+
+    const component = shallow(<Button onClick={() => {}}></Button>);
+
+    expect(component.debug()).toMatchSnapshot();
+  });
+  it("renders a button with text", () => {
+    expect.assertions(1);
+
+    const component = shallow(<Button onClick={() => {}}>Test</Button>);
+
+    expect(component.debug()).toMatchSnapshot();
+  });
+  it("executes passed function when clicked", () => {
+    expect.assertions(1);
+
+    const callBack = jest.fn();
+
+    const wrapper = shallow(<Button onClick={callBack}>Test</Button>);
+
```

Comment:
I normally prefer to keep the 'act' lines together so no need for this space here. Entirely up to you, but either way don't bother removing in this PR.